tests/cephfs/fs_basic_bash_cmds.py

```diff
@@ -0,0 +1,191 @@
+            if wait_for_process(
+                client=nfs_client[0], process_name=nfs_name, ispresent=True
+            ):
+                log.info("ceph nfs cluster created successfully")
+            else:
+                raise CommandFailed("Failed to create nfs cluster")
+            # Create cephfs nfs export
+            if "5.0" in build:
+                nfs_client[0].exec_command(
+                    sudo=True,
+                    cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
+                    f"{nfs_export_name} path={path}",
+                )
+            else:
+                nfs_client[0].exec_command(
+                    sudo=True,
+                    cmd=f"ceph nfs export create cephfs {nfs_name} "
+                    f"{nfs_export_name} {fs_name} path={path}",
+                )
+
+            # Verify ceph nfs export is created
+            out, rc = nfs_client[0].exec_command(
+                sudo=True, cmd=f"ceph nfs export ls {nfs_name}"
+            )
+            if nfs_export_name in out:
+                log.info("ceph nfs export created successfully")
+            else:
+                raise CommandFailed("Failed to create nfs export")
+            # Mount ceph nfs exports
+            nfs_client[0].exec_command(sudo=True, cmd=f"mkdir -p {nfs_mounting_dir}")
+            assert fs_util.wait_for_cmd_to_succeed(
+                nfs_client[0],
+                cmd=f"mount -t nfs -o port=2049 {nfs_server_name}:{nfs_export_name} {nfs_mounting_dir}",
+            )
+            nfs_client[0].exec_command(
+                sudo=True,
+                cmd=f"mount -t nfs -o port=2049 {nfs_server_name}:{nfs_export_name} {nfs_mounting_dir}",
+            )
+            out, rc = nfs_client[0].exec_command(cmd="mount")
+            mount_output = out.split()
+            log.info("Checking if nfs mount is is passed of failed:")
+            assert nfs_mounting_dir.rstrip("/") in mount_output
+            log.info("Creating Directory")
+            out, rc = nfs_client[0].exec_command(
+                sudo=True, cmd=f"mkdir {nfs_mounting_dir}{dir_name}"
+            )
+
+            mount_test_case([clients[0]], nfs_mounting_dir, "nfs_test")
+            return 0
+
+    except Exception as e:
+        log.error(e)
+        log.ereor(traceback.format_exc())
+        return 1
+    finally:
+        log.info("---clean up---------")
+        fs_util.client_clean_up(
+            "umount", fuse_clients=[clients[0]], mounting_dir=fuse_mounting_dir
+        )
+        fs_util.client_clean_up(
+            "umount",
+            kernel_clients=[clients[0]],
+            mounting_dir=kernel_mounting_dir,
+        )
+
+
+def mount_test_case(clients, mounting_dir, filename):
+    """
+    Running commands on mounted directory
+    Args:
+        clients:
+        mounting_dir:
+        filename:
+
+    Returns:
+
+    """
+    log.info("Basic directory operations")
```

Comment:
Can we also include some more operations like : 
ls, cp, mv, rm, ln, more, less,  chmod. 

And these operations should be performed on a directory with a large amount of data or a huge number of small files.